.github/workflows/stencil-nightly.yml

```diff
@@ -98,7 +98,7 @@ jobs:
         # to be the length of the shard array.
         shard: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20]
         totalShards: [20]
-    needs: [build-core]
+    needs: [build-core-with-stencil-nightly]
```

Comment:
Alternatively, we could rename the job itself to `build-core`. The original intent was that when looking at the 'main' CI workflow for Framework, I found myself getting confused (was I looking at `main.yml`, or `stencil-nightly.yml`?). This served as an easier way to differentiate b/w the two. LMK if you'd prefer to make the `s/build-core-with-stencil-nightly/build-core/` change